src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -290,6 +287,11 @@ public void getTriggers(@NonNull Callback<List<Trigger>> callback) {
         getPipe(pipeName).read(getFilter(uri), callback);
     }
 
+    @SuppressWarnings("unchecked")
+    private <T> void savePipe(String pipeName, Object object, Callback<List<T>> callback) {
```

Comment:
Using modified Pipe will be quite troublesome I think.
And using T giving error with calling as "Found org.hawkular.client.android.backend.model.Alert needed Java.lang.String